src/content/configuration/dev-server.md

```diff
@@ -472,6 +473,58 @@ webpack-dev-server --hot-only
 ```
 
 
+## `devServer.http2`
+
+`boolean`
+
+Serve over HTTP/2 using [spdy](https://www.npmjs.com/package/spdy). This option is ignored for Node 10.0.0 and above, as spdy is broken for those versions. The dev server will migrate over to Node's built-in HTTP/2 once Express supports it.
+
+If this option is not explicitly set to `false`, it will default to `true` when using HTTPS. When this option is enabled but the server is unable to serve over HTTP/2, the server defaults to HTTPS.
```

Comment:
```suggestion
If `devServer.http2` is not explicitly set to `false`, it will default to `true` when using HTTPS. When this option is enabled but the server is unable to serve over HTTP/2, the server defaults to HTTPS.
```

Reply:
We already said this option once in previous paragraph, lets keep the context with explicitly naming it here.